### ISSUE TITLE
[Feature] Add an option requiring that space is pressed before triggering a combo.

### ISSUE THREAD

**Issue by xmichelo** [xmichelo/Beeftext#282]
Some users reported interest in new ways to trigger combos (#280). An interesting way would be to require the user to press the spacebar before any combo substitution is performed.

**Comment by xmichelo** [comment]
Here are some test builds. To enable the option, check the box `Preferences ➜ Combos ➜ Combo triggering ➜ Trigger only after spacebar is pressed`.

- [Installer](https://1drv.ms/u/s!AjydspuvJDSngoNIQT6kD_px_QptUg?e=P3ibOm)
- [Portable edition](https://1drv.ms/u/s!AjydspuvJDSngoNHaq90AffF421oEw?e=dXNczx)

**Comment by CosimoMatassini** [comment]
I've checked the test build with [Trigger only after spacebar is pressed]. With this option enabled, the space works as a triggerer but not as an actual character. It should be better to add a space automatically next to the word (where the cursor is) otherwise the spacebar should be pressed twice. What's more, it should be better to work also with special characters like dot (.), comma(,) etc.
Example
bff --> best friend
You're my bff.
If I write bff at the end of a sentence like this with the spacebar option activated the combo won' t be triggered.
I recommend to add this feature or make the spacebar option be set individually on every combo.
If you see TEXT REPLACEMENT in IOS, you should understand what i mean.

**Comment by xmichelo** [comment]
I decided to remove the space character at the end, because in some case you do not want it, and you always have the option to add a space character at the end of your snippet.

However It makes sense to add a preference setting for that. I've created a [new issue](https://github.com/xmichelo/Beeftext/issues/292) for that.

Commas, periods and other punctuation marks can be used in a keyword and as a consequence they cannot be used as a trigger .
